README.md

```diff
@@ -1,14 +1,9 @@
 # <img alt="OpenSK logo" src="docs/img/OpenSK.svg" width="200px">
 
-![markdownlint](https://github.com/google/OpenSK/workflows/markdownlint/badge.svg?branch=stable)
-![pylint](https://github.com/google/OpenSK/workflows/pylint/badge.svg?branch=stable)
-![Cargo check](https://github.com/google/OpenSK/workflows/Cargo%20check/badge.svg?branch=stable)
-![Cargo format](https://github.com/google/OpenSK/workflows/Cargo%20format/badge.svg?branch=stable)
-[![Coverage Status](https://coveralls.io/repos/github/google/OpenSK/badge.svg?branch=stable)](https://coveralls.io/github/google/OpenSK?branch=stable)
-
-*News:*
-
-- 2023-08-24: [PQC paper reference](#Research)
+This branch is unmaintained. It implements the CTAP 2.0 version of OpenSK.
+Please check out the latest versioned branch for maintained code.
```

Comment:
What is the latest versioned branch? The default branch?

Reply:
Yeah, I can call it default branch?

Reply:
That's the Github term yes. When you list branches on Github you see "default" next to it.